### PR TITLE
Place deck/README.md file in a more appropriate place.

### PR DESCRIPTION
**Pull request by carolinar7**
This PR adds the `deck/README.md` into the this repository, for now, in the appropriate place.

/cc @chaodaiG

### CODE REVIEW

**Comment by chaodaiG** [comment]
this looks good, should we also delete the copy from Legacy Snapshot?

**Comment by carolinar7** [comment]
@chaodaiG This PR has been updated to add the abort functionality, and the Legacy README.md was removed!

**Review by chaodaiG — APPROVED** [review]

File: site/content/en/docs/Legacy Snapshot/prow/cmd/deck/README.md [hunk 1]
```diff
@@ -1,50 +0,0 @@
...
---static-files-location=/tmp/deck/var/run/ko/static
---spyglass-files-location=/tmp/deck/var/run/ko/lenses
-```
-
-## Rerun Prow Job via Prow UI
-
-Rerun prow job can be done by visiting prow UI, locate prow job and rerun job by clicking on the ↻ button then clicking `Rerun` button. For prow on github, the permission is controlled by github membership, and configured as part of deck configuration, see [`rerun_auth_configs`](https://github.com/kubernetes/test-infra/blob/0dfe42533307f9733f22d4a6abf08e1df2229fcb/config/prow/config.yaml#L92) for k8s prow.
-
-See example below:
-![Example](./rerun_button.png)
```
Comment (cjwagner) [inline site/content/en/docs/Legacy Snapshot/prow/cmd/deck/README.md:48]: Can we remove this old PNG from the repo as well?

**Review by cjwagner — APPROVED** [review]

File: site/content/en/docs/Legacy Snapshot/prow/cmd/deck/README.md (hunk 1 above)
Comment (carolinar7) [reply site/content/en/docs/Legacy Snapshot/prow/cmd/deck/README.md:48]: The PNG did not exist in the directory, so there was is no need to.

**Pull request merged**